STYLEGUIDE.md

```diff
@@ -0,0 +1,309 @@
+Pronouns are difficult to translate into some languages. They also tend to be gendered, which provides unnecessary bias.
+
+
+### Don’t
+
+“When he opens the app, he will see a loading screen.”
+
+### Do
+
+“When the user opens the app, it will show a loading screen.”
+* * *
+
+## 11. Avoid gerunds (-ing words)
+
+"To" forms of verbs are easier to translate.
+
+### Don’t
+
+“Setting up Amplify UI only takes two lines of code.”
+
+### Do
+
+“You can set up Amplify UI with two lines of code.”
+* * *
+
+## 12. Shorter is better
+
+Short sentences are easier to translate, for both machines and humans! Break clauses into multiple sentences.
+
+### Don’t
+
+“If you are looking to enable triggers for the Storage Category with Amazon S3 & Amazon DynamoDB as Providers, the CLI supports associating Lambda triggers with S3 and DynamoDB events.”
+
+### Do
+
+“You can enable triggers for the Storage Category with Amazon S3 & Amazon DynamoDB as Providers. Amplify supports associating Lambda triggers with S3 and DynamoDB events.”
+* * *
+
+## 13. Lead with location; end with action
+
+When instructing a learner on how to do something, start with the location of the action they’re going to perform so they can navigate to that place, then end with the action they can then use.
+
+### Don’t
+
+“Select **Edge optimized** in the **Endpoint Type** drop-down.”
+
+### Do
+
+“In the **Endpoint Type** drop-down, select **Edge optimized.”**
+* * *
+
+## 14. Use monospace fonts for code, commands, copy-ables; use bold for menus, URLs (clickable), and other UI copy
+
+Code tags and monospace fonts are used to represent code that goes into files or CLI. UI copy, file names, URLs you can click on use **bold.** Remember: monospace is for anything that lives in or is copied to a file or a command line. Everything else is bold.
+
+### Don’t
+
+“To create a new file, go to ‘Menu’ and click on `new file`.”
+
+“In your browser, open `http://localhost:3000/` ”
+
+“Enter the Amplify Library for Swift GitHub repo URL (`https://github.com/aws-amplify/amplify-swift`) into the search bar and hit **Enter**.”
+
+### Do
+
+“To create a new file, go to **Menu > New File.”**
+
+“In your browser, open **http://localhost:3000/”**
+
+“Enter the Amplify Library for Swift GitHub repo URL (`https://github.com/aws-amplify/amplify-swift`) into the search bar and hit **Enter**.”
+* * *
+
+## 15. Use carets to help users navigate menus; use “select” instead of “click”
+
+* For instructions for navigating a menu, use carets to indicate the hierarchy of the menu items. This is concise and easy to follow.
+* Tell people to “select” things instead of “click on them”. Not everyone uses a mouse—many people tab or touch items.
+
+
+### Don’t
+
+“To create a new file, go to ‘Menu,’ select ‘Art,’ and click on ‘new file’.”
+
+“Click on ‘upload image.’”
+
+### Do
+
+“To create a new file, go to **Menu > Art > New File****.”**
+
+“Select **upload image.”**
+* * *
+
+## 16. Bracket <replacement text> in codeblocks
```

Comment:
This guidance will cause friction writing markdown docs, since Markdown usually interprets text between angle brackets as HTML tags. Those will usually be invisible to readers (see screenshot for an example using this rule's rendered view)

![image](https://user-images.githubusercontent.com/227982/201136693-ee8be9ae-4056-4ae8-a468-50431b1e84d3.jpeg)

Reply:
Too true. Though, this was suggested by some of our own engineers. Is there a more standard convention that won't conflict?